services/provider/server/server.go

```diff
@@ -77,6 +77,7 @@ const (
 	monSecret                        = "rook-ceph-mon"
 	volumeReplicationClass5mSchedule = "5m"
 	mirroringTokenKey                = "rbdMirrorBootstrapPeerSecretName"
+	ExternalClassLabelKey            = "storageclass.kubernetes.io/is-external"
```

Comment:
Let's move it to storagecluster/utils?

Reply:
i cant see such file, can you be specific on which file among this?
https://github.com/red-hat-storage/ocs-operator/tree/main/controllers/util

Reply:
https://github.com/red-hat-storage/ocs-operator/blob/main/controllers/util/k8sutil.go
Should be a good place